test/e2e/swaps/shared.js

```diff
@@ -8,6 +10,17 @@ const ganacheOptions = {
   ],
 };
 
+const driverOptions = {
+  timeOut: 20000,
+};
+
+const withFixturesOptions = {
+  fixtures: new FixtureBuilder().build(),
+  ganacheOptions,
+  driverOptions,
+  failOnConsoleError: false,
```

Comment:
is failOnConsoleError still needed?

Reply:
Yes, we are still getting React errors with some tests:
![image](https://user-images.githubusercontent.com/80175477/199076585-20cce9d3-9f3a-444d-8975-8774778fea01.png)

I can try to clean up those errors and then this won't be needed.